tests/rsvp_v2_integration/RSVP/V2/Repositories/Attendee_Repository_Test.php

```
@@ -358,6 +366,104 @@ public function test_delete_attendee_returns_event_id(): void {
+			'purchaser_user_id'    => 0,
+			'purchaser_full_name'  => 'Test Purchaser',
+			'purchaser_first_name' => 'Test',
+			'purchaser_last_name'  => 'Purchaser',
+			'purchaser_email'      => 'attendee@example.com',
+		];
+
+		/** @var Order $orders */
+		$orders = tribe( Order::class );
+		$order  = $orders->create_from_cart( tribe( Gateway::class ), $purchaser, Constants::TC_RSVP_TYPE );
+
+		$orders->modify_status( $order->ID, Pending::SLUG );
```

PHPStan (2.2.8)

[error] Cannot access property $ID on WP_Post|false.


(property.nonObject)